Dockerfile

```diff
@@ -93,7 +94,7 @@ WORKDIR /opt/app-root/app
 USER 1001
 
 ENV LUA_CPATH "./?.so;/usr/lib64/lua/5.1/?.so;/usr/lib64/lua/5.1/loadall.so;/usr/local/lib64/lua/5.1/?.so"
-ENV LUA_PATH "/usr/lib64/lua/5.1/?.lua;/usr/local/share/lua/5.1/?.lua;/usr/local/share/lua/5.1/*/?.lua;"
+ENV LUA_PATH "/usr/lib64/lua/5.1/?.lua;/usr/local/share/lua/5.1/?.lua;/usr/local/share/lua/5.1/*/?.lua;;"
```

Comment:
this change is expected? why?

Reply:
`;;` can be used to stand for the original path, and the reason for this is because lua-resty-openssl import module with `init.lua` file.

For example:
when we call `local pkey = require ('resty.openssl.pkey')` . That will then call `require "resty.openssl.include.x509"`.

* Old path, openresty will try to find `/usr/local/share/lua/5.1/resty/openssl/include/x509.lua` which non exist
* New path (with `;;`) openresty will also search for `init.lua` file, ie `/usr/local/share/lua/5.1/resty/openssl/include/x509/init.lua`

Reply:
Thanks for the explanation! Good to know!

Reply:
> Verification steps worked, but the patch is defined on dev-environments/https-proxy-upstream-tlsv1.3 and should be dev-environments/upstream-tlsv1.3

With recent changes in #1499, mtls can also work when forwarding requests through a proxy. I will update the verification steps to `dev-environments/upstream-tlsv1.3` and merge this one.

Reply:
Ah sorry, just realized that I gave the wrong instruction the in the verification steps.

Reply:
Fixed